### DISCUSSION TITLE
Cannot install rust

### DISCUSSION THREAD

**Discussion by RumuriReiji** [termux/termux-app#3326] [General]
Anyone has solution?
![Screenshot_20230331-231524_Termux](https://user-images.githubusercontent.com/86666160/229174932-033475b1-7ae9-447c-8c68-c8815ff7a804.jpg)

**Comment by sk0kanik** [comment]
termux/termux-packages#15851

**Reply by ghost** [reply]
> [termux/termux-packages#15851](https://github.com/termux/termux-packages/issues/15851)

Since that issue is Closed, I send the message here: [TermuxArch](https://github.com/TermuxArch/TermuxArch) downloads ArchLinuxARM for Termux, and the repository of ArchLinuxARM still uses 15.0.7 version of llvm-libs. I tested LLVM16 in Termux, [zeroize_derive](https://crates.io/crates/zeroize-derive) failed to compile, but with TermuxArch, zeroize_derive succeeded to compile. It looks that LLVM16 is incompatible with Rust.
